drivers/hmis/app/models/hmis/form/definition.rb

```diff
@@ -591,7 +596,8 @@ def self.infer_cded_field_type(item_type)
     when 'CURRENCY'
       'float'
     else
-      raise "unable to determine cded type for #{item_type}"
+      'string'
```

Comment:
Sorry this is an artifact from debugging, I'll remove it after getting to the bottom of the issues with the cded generating rake task.

Reply:
(After this, the next error is
```
rake aborted!
ActiveRecord::NotNullViolation: PG::NotNullViolation: ERROR:  null value in column "key" of relation "CustomDataElementDefinitions" violates not-null constraint
DETAIL:  Failing row contains (714, Hmis::Hud::Client, null, boolean, null, See DOB, f, 3, 3800b2c8fe534842a7a38740d5d3aaf5, 2024-07-08 14:28:32.607318, 2024-07-08 14:28:32.607318, null, f, null).
```
)